bundles/edu.kit.kastel.sdq.eclipse.grading.view/src/edu/kit/kastel/eclipse/grading/view/assessment/AssessmentTab.java

```diff
@@ -33,6 +33,8 @@ public void setAssessmentInProgress(boolean courseSelected, boolean examSelected
 		btnSubmit.setEnabled(assessmentStarted);
 		btnCloseAssessment.setEnabled(assessmentStarted);
 
+		btnRerunAutograder.setEnabled(assessmentStarted);
```

Comment:
Enable this button only if configuration of the autograder is set in the properties.

Reply:
Fixed